src/controller/friend.controller.ts

```diff
@@ -94,10 +98,14 @@ export const getFriendRequests = async (
   res: NextApiResponse
 ) => {
   try {
-    const { userId } = req.query;
+    const { userId, accepted } = req.query;
     if (typeof userId !== "string") {
       throw REQUEST_QUERY_ERROR;
     }
+    // 친구 목록 GET
+    if (typeof accepted === undefined) {
+    }
+    // 친구 요청 목록 GET
```

Comment:
친구 목록 GET은 나중에 구현하려는 것인가요? 그렇다면 TODO 주석이 있는게 좋아보이네요.
```suggestion
    // 친구 목록 GET
    if (typeof accepted === undefined) {
       // TODO: 친구 목록 GET 구현하기
    }
    // 친구 요청 목록 GET
```

Reply:
지금 만들거라서 안 고치고 바로 진행하겠습니다 !